sys/random/mersenne.c

```diff
@@ -129,7 +133,12 @@ uint32_t genrand_uint32(void)
     y ^= (y << 7) & 0x9d2c5680UL;
     y ^= (y << 15) & 0xefc60000UL;
     y ^= y >> 18;
+
+#ifdef MODULE_ENTROPY
+    return y + entropy;
```

Comment:
> hey had performed a larger study on common sources of 'entropy' (typing, perturbed wireless signals, other hardware fluctuations) and results were disastrous: no reliable unpredictability nowhere.

I'd like to read that study.

Apart from that, I think we have several levels of needed 'randomness'.

- "bootup-safe" -> 1000(*n) nodes booting up at the same time will not have the same IP address if that is derived from our "random number generator"

- "network-safe" -> "random" back-off will not collide between nodes

- "crypto-safe" -> the higher gods of crypto math consider it "random" enough for safe crypto

Let's solve them one at a time.

This PR might help with the letter two if used correctly. And it adds only 15 lines of code.